<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OptionsWidget</class>
 <widget class="QWidget" name="OptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="timelineTab">
      <attribute name="title">
       <string>Timeline</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Color</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QRadioButton" name="colorByPerson">
            <property name="text">
             <string>By Person</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="colorGrayscale">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>Grayscale</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="colorByHypothesis">
            <property name="text">
             <string>By Hypothesis</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Fixed</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="colorOnlyCone">
              <property name="text">
               <string>Only the Cone phenomenon</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="colorByStep">
            <property name="text">
             <string>By ISLE Step</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_4">
         <property name="title">
          <string>Graph size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_7">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Time scale (X)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="xScaleSlider">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="value">
             <number>30</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Vertical scale (Y)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="yScaleSlider">
            <property name="value">
             <number>40</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Split timeline at</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QSpinBox" name="splitMinutes">
              <property name="suffix">
               <string> minutes</string>
              </property>
              <property name="minimum">
               <number>10</number>
              </property>
              <property name="maximum">
               <number>120</number>
              </property>
              <property name="singleStep">
               <number>5</number>
              </property>
              <property name="value">
               <number>35</number>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showJudgment">
         <property name="text">
          <string>Show Judgment Markers</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="showGridLines">
         <property name="text">
          <string>Show grid lines</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="cycleTab">
      <attribute name="title">
       <string>Cycle</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="coloredStepsCheck">
         <property name="text">
          <string>Use colored steps</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Connections</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QRadioButton" name="transitionsRadioButton">
            <property name="text">
             <string>Transitions</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="overlapRadioButton">
            <property name="text">
             <string>Overlap</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>Graph size</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_6">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Cycle radius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="cycleRadiusSlider">
            <property name="minimum">
             <number>50</number>
            </property>
            <property name="maximum">
             <number>400</number>
            </property>
            <property name="singleStep">
             <number>5</number>
            </property>
            <property name="pageStep">
             <number>20</number>
            </property>
            <property name="value">
             <number>200</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Step circle size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="stepSizeSlider">
            <property name="maximum">
             <number>50</number>
            </property>
            <property name="pageStep">
             <number>5</number>
            </property>
            <property name="value">
             <number>15</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Line thickness</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="thicknessSlider">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Arrow size</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="arrowSizeSlider">
            <property name="value">
             <number>50</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Arrow position</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="arrowPositionSlider">
            <property name="value">
             <number>70</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Arrow center point</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSlider" name="arrowPointSlider">
            <property name="maximum">
             <number>15</number>
            </property>
            <property name="pageStep">
             <number>3</number>
            </property>
            <property name="value">
             <number>5</number>
            </property>
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="expertivityTab">
      <attribute name="title">
       <string>Expertivity</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_8">
       <item>
        <widget class="QPushButton" name="weightsButton">
         <property name="text">
          <string>Configure Weights</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Parts</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numberOfParts">
           <property name="minimum">
            <number>2</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="value">
            <number>10</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>transitionsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>arrowPositionSlider</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>93</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>116</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>transitionsRadioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>label_4</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>183</x>
     <y>106</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>365</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorByHypothesis</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorOnlyCone</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>103</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>133</x>
     <y>159</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>colorByPerson</sender>
   <signal>toggled(bool)</signal>
   <receiver>colorGrayscale</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>90</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
